members/models.py

```diff
@@ -0,0 +1,56 @@
+from core.models import *
+
+
+# Create your models here.
+class Member(models.Model):
+    """
+    The member model.
+
+    It contains:
+        -an optional reference to the user if it already exists.
+        -the semester the membership was sold in
+            (and for ordinary members also the semester it's valid for)
+        -if the member got a lifetime or honorary membership.
+        -the date the membership was sold
+        -who sold it
+        -name of the member
+        -email address that can be blank
+
+    Can only be one member with the same name and email address per semester.
+    """
+    user = models.ForeignKey(User, null=True, blank=True, related_name='user')
+    semester = models.ForeignKey(Semester)
+    lifetime = models.BooleanField(blank=False, help_text='Lifetime member')
```

Comment:
Heller sette default=False på disse to? 

blank=False på en boolean gir ikke helt mening. Tror den enforcer True uansett, så kan nok fjernes